src/dynarec/rv64/dynarec_rv64_0f.c

```diff
@@ -151,6 +151,16 @@ uintptr_t dynarec64_0F(dynarec_rv64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 FMVD(v1, v0);
             }
             break;
+        case 0x13:
```

Comment:
@ptitSeb Just a question to ask, I originally implemented this opcode as:
```
case 0x13:
    INST_NAME("MOVLPS Ex,Gx");
    nextop = F8;
    GETGXSD(v0);
    GETEXSD(v1, 0);
    if (v0!=v1) FMVD(v1, v0);
    if(!MODREG) SMWRITE2();
    break;
```

But the game cannot run correctly, do you know what's wrong with this implementation?

Reply:
I don't don't see anything wrong here. not sure the use of "SD" variant is a good thing for a "PS" opcode tho.

Reply:
Hmm, it's strange indeed, if we use this implementation, the little people on the landing page will turn blue..

![2023-04-20 21 27 29](https://user-images.githubusercontent.com/15257961/233380992-39bd44ad-f2e8-48b4-87cd-c6fb0b6f5232.jpg)

Reply:
Ok, I'll try that.

Reply:
Oh, I think it is because `GETEXSD` cannot be used as a destination when ed is a memory address.

https://github.com/ptitSeb/box64/blob/84b0ee2284598a65ef7f727bb27b8c1527e8b5a3/src/dynarec/rv64/dynarec_rv64_helper.h#L301-L309

Reply:
Ah, well, yeah, there must be some "EXBACK" or something like that indeed!